lte/gateway/configs/pipelined.yml_prod

```diff
@@ -128,6 +128,10 @@ redis_enabled: true
 # Logs grpc payload content
 magma_print_grpc_payload: false
 
+# MTR iface IP
+mtr_ip: 10.1.0.1
```

Comment:
This is duplicating IP address value from network interface file. Rather than specifying this as config point, can we read directly from interface ?